code/lib/preview-web/src/PreviewWeb.test.ts

```diff
@@ -2162,10 +2169,11 @@ describe('PreviewWeb', () => {
 
           // Wait three ticks without resolving the play function
           await waitForSetCurrentStory();
-          await waitForSetCurrentStory();
-          await waitForSetCurrentStory();
+          // We can't mock setTimeout for this test, due to waitForSetCurrentStory hack,
+          // So give some (real) time for the reload to be called
+          await new Promise((r) => setTimeout(r, 100));
```

Comment:
This might turn out to be inconsistent?

Reply:
Yeah I'm not sure.  Previously we were using a `setImmediate` to accomplish the same thing, so I _think_ we'll be safe, but I banged my head on this for half a day and couldn't come up with a better solution.

Reply:
Hmm, do we want to disable this test for a second and I can take a look at it next week? It feels like it's getting pretty brittle and might need to be rethought.

Reply:
Or we could leave as is in this PR, and I can take a look then too.

Reply:
It might be interesting to re-address once we get to jest 29, but I don't think anything will change.  

I think it would be great if you wanted to take a look at it when you've got some time, but it might be worth leaving the test running and skipping it only if we find out that it's flakey.  I don't feel strongly either way.

Reply:
Yes, let's leave it for now, I will make a note.